product-features/farming/key-concepts.md

```diff
@@ -0,0 +1,102 @@
+---
+description: >-
+  All you need to know for successfully farming on ALEX Lab, from farm basics to
+  dashboard metrics!
+---
+
+# Key concepts
+
+## Farm basics
+
+Yield farming works very similar to standard staking, with the key difference being that the tokens you stake are LP tokens. As in traditional staking, you lock up your tokens for a certain period (measured in cycles) and earn rewards over time. After each cycle, you will have rewards available to harvest.
```

Comment:
Yield farming works in a very similar way to standard staking, with the key.....